storybook/pages/link.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/594a9a3356.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../css/storybook.css">
+    <title>Link Component</title>
+</head>
+<body>
+
+    <span class="title">Span Links:</span>
+    <span class="link">
+        <a href="#" class="link__content">Sample Text</a>
+    </span>
+
+    <span class="link link--with-arrow">
+        <a href="#" class="link__content">With Icon</a>
+        <i class="link__icon fa-solid fa-caret-right"></i>
+    </span>
+
+    <span class="title">Li Links:</span>
+    <li class="link">
+        <a href="#" class="link__content">Default text</a>
+    </li>
+
+    <li class="link link--with-arrow">
+        <a href="#" class="link__content">Very very very long text with icon to show behaviour</a>
```

Comment:
![image](https://github.com/user-attachments/assets/0c7bf60c-52b6-40fc-b665-005bf6c0aa97)
I like that